dotcom-rendering/src/web/components/Callout/CalloutDescription.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import { textSans, space, body } from '@guardian/source-foundations';
+import { decidePalette } from '../../lib/decidePalette';
+
+const descriptionStyles = (format: ArticleFormat) =>
+	css`
+		a {
+			color: ${decidePalette(format).text.richLink};
+			border-bottom: 1px solid ${decidePalette(format).text.richLink};
+			text-decoration: none;
```

Comment:
We want the accent colours to change based on the pillar, hence we use decidePalette here